srv/modules/runners/replace.py

```diff
@@ -147,15 +149,45 @@ def minion_profile(minion):
     """
     files = __salt__['push.organize']()
 
+
     yaml_file = 'stack/default/ceph/minions/{}.yml'.format(minion)
     if yaml_file in files:
         for filename in files[yaml_file]:
             if os.path.exists(filename):
-                print("Renaming minion {} profile".format(minion))
-                os.rename(filename, "{}-replace".format(filename))
+                try:
+                    print("Renaming minion {} profile".format(minion))
+                    os.rename(filename, "{}-replace".format(filename))
+                    _insert_replace_tag(grains, minion, osds, "{}-replace".format(filename))
+                except:
+                    log.error("Failed to rename minion {} profile".format(minion))
+                    os.rename("{}-replace".format(filename), filename)
+
     return ""
 
 
+def _strip_enddigits(string):
```

Comment:
Doe this do somehting different then ```string.rstrip('0123456789')```? If I'm not missing anything this method is unnecessary.

Reply:
No, this is a case of "do before properly looking if there is an built-in that does the same" :laughing: I'll adapt the code.